### DISCUSSION TITLE
Unsigned debian repo stops cnspec install script from running

### DISCUSSION THREAD

**Discussion by yzl** [mondoohq/cnquery#803] [Q&A] [answered]
When I run (via github action running on ubuntu) the cnspec install script, I get an error message that 'https://releases.mondoo.com/debian stable InRelease' is unsigned. The same action worked this morning Pacific time. Can someone suggest a workaround/fix? I'm running in a github action, so my ability to hack the script is limited.

```
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Hit:1 http://azure.archive.ubuntu.com/ubuntu jammy InRelease
Hit:2 http://azure.archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:3 http://azure.archive.ubuntu.com/ubuntu jammy-backports InRelease
Hit:4 http://azure.archive.ubuntu.com/ubuntu jammy-security InRelease
Hit:5 https://packages.microsoft.com/ubuntu/22.04/prod jammy InRelease
Get:6 https://releases.mondoo.com/debian stable InRelease [14.0 kB]
Hit:7 https://ppa.launchpadcontent.net/ubuntu-toolchain-r/test/ubuntu jammy InRelease
Err:6 https://releases.mondoo.com/debian stable InRelease
  The following signatures were invalid: EXPKEYSIG AC69C65100E1C42B Mondoo Inc <security@mondoo.io>
Reading package lists...
W: GPG error: https://releases.mondoo.com/debian stable InRelease: The following signatures were invalid: EXPKEYSIG AC69C65100E1C42B Mondoo Inc <security@mondoo.io>
E: The repository 'https://releases.mondoo.com/debian stable InRelease' is not signed.
The Mondoo cnspec install script encountered a problem. For assistance, please join our community on GitHub Discussions.
```

**Comment by benr** [comment, marked as answer]
Hello @yzl !  Please re-run your action, it should work just fine now.

**Reply by yzl** [reply]
HI @benr! I kicked off a run of my action just now, and it failed again with the same error.

**Reply by benr** [reply]
I'm still seeing some cache effect from the CDN, I've forced a cache flush to speed it up.  If it persists please let me know.

**Reply by yzl** [reply]
In case it's useful, I ran it in two different repos: it worked in one, and got the repo signing error in the other.

**Reply by yzl** [reply]
ok, it's working now in the repo where it wasn't before, so it looks like the cache flush helped. Thank you!
